report.sh

```diff
@@ -130,23 +130,23 @@ case $opcao in
 			TIME=3
 			echo " "
```

Comment:
Bash expects content between double quote to have to be intepreted. So if you have nothing to interpret, it's cleaner to use simple quote:
``echo ' '`
This way the interpreter won't even try to process the content and the reader will know what was your intention here.